### PR TITLE
Adds the ability to choose the App Service plan

### PR DESCRIPTION
**Pull request by MCKLMT**
### Changelog

* Adds the ability to choose the App Service plan for the Logic Apps (fix issue #2243)

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@MCKLMT__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (mimott). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: 201-alert-to-slack-with-logic-app/azuredeploy.json [hunk 1]
```diff
@@ -40,13 +78,20 @@
         },
         {
             "type": "Microsoft.Logic/workflows",
-            "apiVersion": "2016-06-01",
+            "apiVersion": "2015-02-01-preview",
```
Comment (gatneil) [inline 201-alert-to-slack-with-logic-app/azuredeploy.json]: Why did we go backwards in time with the API? Is there no non-preview API we can use?

**Comment by MCKLMT** [comment]
@gatneil There are only two versions of this API. When I try to deploy with the lastest, I received an error explaining that deploying in an App Service plan is deprecated in this API version.
This is why I'm using the previous one.

**Comment by gatneil** [comment]
Looks good to me; merging :)

**Pull request merged**